### PR TITLE
feat(Recipes): New Recipe

### PR DESCRIPTION
**Pull request by frivajica**
Adding a new recipe to use as boilerplate for a Dockerized API with auth (JWT) and PostgreSQL.

### CODE REVIEW

File: auth-docker-postgres-jwt/go.mod [hunk 1]
```diff
@@ -0,0 +1,41 @@
+module app
```
Comment (gaby) [inline auth-docker-postgres-jwt/go.mod:1]: Add an entry in .github/dependabot.yml to track this file

**Review by gaby — CHANGES_REQUESTED** [review]

File: auth-docker-postgres-jwt/go.mod (hunk 1 above)
Comment (frivajica) [reply auth-docker-postgres-jwt/go.mod:1]: Sure, working on it today

Comment (frivajica) [reply auth-docker-postgres-jwt/go.mod:1]: Done 👍

**Review by gaby — APPROVED** [review]

**Comment by frivajica** [comment]
Thank you, but I don't have write access to merge it

**Pull request merged**